mono/mini/mini.h

```diff
@@ -783,16 +783,34 @@ struct MonoCallInst {
 	MonoInst *out_args;
 	MonoInst *vret_var;
 	gconstpointer fptr;
+	MonoJitICallId jit_icall_id;
 	guint stack_usage;
 	guint stack_align_amount;
+	regmask_t used_iregs;
+	regmask_t used_fregs;
+	GSList *out_ireg_args;
+	GSList *out_freg_args;
+	GSList *outarg_vts;
+	CallInfo *call_info;
+#ifdef ENABLE_LLVM
+	LLVMCallInfo *cinfo;
+	int rgctx_arg_reg, imt_arg_reg;
+#endif
+#ifdef TARGET_ARM
+	/* See the comment in mini-arm.c!mono_arch_emit_call for RegTypeFP. */
+	GSList *float_args;
+#endif
+	// Bitfields are at the end due to an unexplained problem with C++ and LLVM.
```

Comment:
It is explained and fixed actually.

Reply:
1. One of the files wasn't compiling as C++. automake typo, sort of.
2. Fixed twice: the compilation was fixed, and the ifdef around MonoJitCallId was removed, so it'll work even with that mistake.

Reply:
oops.  so should we revert the commit?

Reply:
Comment was reworked in https://github.com/mono/mono/commit/c32414966c76a10741949d4f40e0dc737753e44f.